main.go

```diff
@@ -135,7 +135,7 @@ func main() {
 
 	commandSet := jhanda.CommandSet{}
 	commandSet["activate-certificate-authority"] = commands.NewActivateCertificateAuthority(api, stdout)
-	commandSet["apply-changes"] = commands.NewApplyChanges(api, logWriter, stdout, applySleepSeconds)
+	commandSet["apply-changes"] = commands.NewApplyChanges(api, api, logWriter, stdout, applySleepSeconds)
```

Comment:
is there a particular reasoning behind using two api service instead of extending `applyChangesService` interface to include `ListStagedPendingChanges()` method?

Reply:
Hi @fredwangwang there was no specific reason, we were just trying to do it in a way that would be least intrusive if that answers your question?